Docs/source/maintenance/performance_tests.rst

```diff
@@ -62,22 +62,25 @@ Then, in ``$AUTOMATED_PERF_TESTS``, create a file ``run_automated_performance_te
    # Make sure all dependencies are installed and loaded
    cd $HOME
    module load python/3.8.10
+   module load freetype/2.10.4     # matplotlib
    module load openblas/0.3.5-omp
    export BLAS=$OLCF_OPENBLAS_ROOT/lib/libopenblas.so
    export LAPACK=$OLCF_OPENBLAS_ROOT/lib/libopenblas.so
    python3 -m pip install --user --upgrade pip
    python3 -m pip install --user virtualenv
    python3 -m venv $HOME/sw/venvs/warpx-perftest
    source $HOME/sw/venvs/warpx-perftest/bin/activate
-   python3 -m pip install --upgrade pip
-   python3 -m pip install --upgrade cython
-   python3 -m pip install --upgrade numpy
-   python3 -m pip install --upgrade markupsafe
-   python3 -m pip install --upgrade pandas
-   python3 -m pip install --upgrade matplotlib==3.2.2
-   python3 -m pip install --upgrade bokeh
-   python3 -m pip install --upgrade gitpython
-   python3 -m pip install --upgrade tables
+   # Do the above and following steps once, before
+   # submission. They take to long to do every time.
```

Comment:
Small typo. And a question: about the steps "above" that one should do only once before submission, which ones are they? I mean, the "following" ones are the ones below commented out, but I think I might not understand the "above" ones. Should we just place the comment before all the steps that need to be done only once (instead of in the middle)? Sorry if I misunderstood something. 
```suggestion
   # Do the above and following steps once, before
   # submission. They take too long to do every time.
```

Reply:
Good point, will clarify :)